packages/shared/lib/core/profile/stores/coin-type.store.ts

```diff
@@ -0,0 +1,5 @@
+import { COIN_TYPE } from '@core/network/constants'
+import { activeProfile } from './active-profile.store'
+import { derived } from 'svelte/store'
+
+export const coinType = derived([activeProfile], ([$activeProfile]) => COIN_TYPE[$activeProfile.networkProtocol])
```

Comment:
For mocking purposes, it's better to not export stores that do not need to be reactive and create a `getCoinType` function that returns a string, since that's how you use it most of the time.